File: Python/1010.py
product_1 = input().split(" ")[1:]
product_2 = input().split(" ")[1:]
total = (int(product_1[0]) * float(product_1[1])) + (
    int(product_2[0]) * float(product_2[1])
)
print("VALOR A PAGAR: R$ {:.2f}".format(total))

File: JavaScript/1010.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var x = lines.shift();
var y = lines.shift();

x = x.split(" ");
y = y.split(" ");

console.log("VALOR A PAGAR: R$ " + (x[1] * x[2] + y[1] * y[2]).toFixed(2));
